docs/src/domains/cdn/cloudflare.md

```diff
@@ -4,12 +4,12 @@ weight: 2
 sidebarTitle: "Cloudflare"
 toc: false
 aliases:
-  - "/golive/steps/cloudflare.html"
+  - "/domains/in-depth/cloudflare.html"
 ---
 
 One of the main features that a modern DNS provider needs to have in order to work well with Platform.sh is colloquially known as "Cname Flattening".  This solves the problem of being able to point your "root domain" (example.com) to a domain name (CNAME) rather than an IP address (A record).  [This post](https://blog.cloudflare.com/introducing-CNAME-flattening-rfc-compliant-cnames-at-a-domains-root/) explains it well.
 
-In order to correctly point DNS to your Platform.sh project, you'll need at the very least the master environment CNAME, in other words the domain of your site before you add a custom domain on the management console for that project (or otherwise in the CLI).  This is the value you would get from [Step 2 of the pre-launch checklist](/golive/checklist.md).
+In order to correctly point DNS to your Platform.sh project, you'll need at the very least the master environment CNAME, in other words the domain of your site before you add a custom domain on the management console for that project (or otherwise in the CLI).  This is the value you would get from [Step 2 of the before start checklist](/domains/checklist.md).
```

Comment:
```suggestion
In order to correctly point DNS to your Platform.sh project, you'll need at the very least the master environment CNAME, in other words the domain of your site before you add a custom domain on the management console for that project (or otherwise in the CLI).  This is the value you would get from [Step 2 of the pre-launch checklist](/domains/checklist.md).
```